webclient/app/src/app/features/apps/entities/entityDialog/EntityDialog.jsx

```diff
@@ -79,7 +79,7 @@ function EntityDialog(props) {
 
     function getTargetEntities() {
         let newTargetEntities = [];
-        if (entities?.length > 1) {
+        if (entities?.length => 1) {
```

Comment:
```suggestion
        if (entities?.length >= 1) {
```

Reply:
Yes, if you add a second entity and a relation to the first entity, it is needed.